bin/infobot

```diff
@@ -194,28 +190,26 @@ print "Loading Loader: ";
 print "".(($bot->load("Loader"))? "OK" : "NOT OK");
 print "\n";
 
-
-# Then set the keys the plugins
+# Set module parameters from config. Load module if necessary.
 foreach my $key (keys %conf) {
     next if $key eq 'store';
     next unless ref $conf{$key} eq 'HASH';
-    my $mod = $bot->load($key);
-
-
-    unless (defined $mod) {
-        print "We haven't loaded a module $key - config values will be ignored\n";
-        next; # should we 'next' now? bah, wait for people to compain 
+    my $mod;
+    unless ($mod = $bot->module($key)) {
+        $mod = $bot->load($key);
+        if (!defined $mod) {
+            print "We haven't loaded a module $key - config values will be ignored\n";
+            next;
+        }
     }
-
     foreach my $sub_key (keys %{$conf{$key}}) {
         my $val = $conf{$key}->{$sub_key};
-        $mod->set("user_${sub_key}", $val);
+        $mod->set("${sub_key}", $val);
```

Comment:
Since you've played with this more recently than I have I'm willing to accept that changing this is the right thing to do but I must have originally written it as ```"user_${sub_key}"``` for a reason. 

I'd be more comfortable if we could figure out why that was